packages/@react-aria/overlays/src/DismissButton.tsx

```diff
@@ -41,7 +41,7 @@ export function DismissButton(props: DismissButtonProps) {
   };
 
   return (
-    <VisuallyHidden>
+    <VisuallyHidden style={{marginBlockStart: '-10px'}}>
```

Comment:
Open to opinions: Maybe we should open up the api so user provided styles to the DismissButton are passed to the VisuallyHidden component? Just a bit concerned that the `-10px` is making an assumption about the positioning of

Reply:
@LFDanLu Yeah, the `-10px` is based on the height of the hit target for the VisuallyHidden element, so if that were to change...

Reply:
See https://github.com/adobe/react-spectrum/commit/9ee38ed47fd6f36f9f351f83410fb488ae64e5be